style_guide.md

```diff
@@ -1,26 +1,100 @@
-# BODS Style Guide - #draft
+# BODS Style Guide - #WIP
+
+This style guide is a draft to be discussed and not yet implemented across the BODS schema and documentation (as of May 2019, BODS v0.2). It can be used to guide editing and content writing.
+## Common Conventions
+
+### Readability 
+We try to write clear, concise documentation, using plain English where possible. 
+
+These tools can help with this:
+- [Hemingway editor](https://hemingwayapp.com/)
+- [Grammarly](https://www.grammarly.com/)
+- [Readability tools](https://www.webfx.com/tools/read-able/)
+
+### Spelling 
+Use English spelling:
+- 'organisation' not 'organization'
+- 'standardised' not 'standardized'
+- 'modelling' not 'modeling'
+
+### Word choice
+- use 'person' instead of using 'individual' as a noun
```

Comment:
@kd-ods how about this? 
- use 'person' instead of using 'individual' as a noun. You can also use 'natural person' to avoid confusion with 'legal person'
- use 'entity' to refer to legal entities or use specific descriptors like 'trust' where relevant
- do not use 'person' or 'legal person' to refer to entities, 'person' should be used for natural persons only

Reply:
I think the first two bullets are good. I'd scrap the third one since there are likely to be instances where we need to be clear about how entities with or without legal personhood should be classified; i.e. we'll need to refer to 'legal persons' as a concept.

Reply:
yeah that makes sense - have edited that now :)